### ISSUE TITLE
All of my sites and projects have disappeared

### ISSUE THREAD

**Issue by ltseng** [techmatters/terraso-mobile-client#681]
### App Version

59

### Account

lilly.tseng@gmail.com

### Platforms

- [X] iOS
- [X] Android
- [ ] iOS simulator
- [ ] Android emulator

### Description

When I launch the app, all of the sites and projects I have created previously do not appear, and my projects and sites lists say that they are empty. If I create any sites or projects in that session, close the app, and relaunch it, they also disappear. This occurs on Android and iOS (iPhone and iPad) intermittently for my Gmail account.

### Steps to Reproduce

Launch the app

### Expected behavior

Previously created sites and projects are loaded

### Actual behavior

Previously created sites and projects are not loaded

**Comment by ltseng** [comment]
My sites and projects started randomly loading again! I'm going to close this one for now, but if it comes back, I will reopen.

**Comment by ltseng** [comment]
I spoke too soon, they have disappeared again on Android, build 60 😭 

tagging @david-code and @paulschreiber for awareness

**Comment by ltseng** [comment]
The problem is still persisting in Build 72 on iOS and Android. I've recorded this new video on iPad that shows an interesting new error when I create a site, followed by the app crashing - `Error saving Soil ID information`


https://github.com/techmatters/terraso-mobile-client/assets/2730009/ef8fde6e-9e5b-4200-9439-168989863aa6

**Comment by CourtneyLee333** [comment]
I've just encountered this issue and in addition, it seems that the app is possibly caching account info. 

Note that I'm using two different accounts in the app which both have had sites and projects created in them. Before starting this screen recording I signed out of the app and stopped it.

Here's what the video shows:

- First, when I log into my techmatters account my sites and projects lists show empty.
- I log out, then log into my gmail account where I see a big list of sites.
- I log out and back into my techmatters account where I see the same list of sites from my gmail account.
- I check the project > team to see that it is displaying my photo from my gmail account instead of my techmatters account.

[Here is the screen recording. ](https://drive.google.com/file/d/1JACCjqbd1ANwob75o_yemMcrPRm73Coa/view?usp=sharing)
